src/libstd/primitive_docs.rs

```diff
@@ -891,9 +895,9 @@ mod prim_usize { }
 /// A reference represents a borrow of some owned value. You can get one by using the `&` or `&mut`
 /// operators on a value, or by using a `ref` or `ref mut` pattern.
 ///
-/// For those familiar with pointers, a reference is just a pointer that is assumed to not be null.
-/// In fact, `Option<&T>` has the same memory representation as a nullable pointer, and can be
-/// passed across FFI boundaries as such.
+/// For those familiar with pointers, a reference is just a pointer that is assumed to be
```

Comment:
This could be read as the full list of differences between raw pointers and references, but surely we require more? I'm thinking wrt. wrt. https://github.com/rust-lang/unsafe-code-guidelines/issues/77 here.

Reply:
Hm, not sure. As you see I didn't write this part. It surely must also not be dangling though.

Reply:
Should we, for now, add a cop-out using "including" to make it seem less like "the definitive list"?

Reply:
I think the last version lists all restrictions that are present at https://github.com/rust-lang-nursery/nomicon/pull/149, so it better be complete.

Reply:
Yep; looks good.